docs/setup/implementation_setup_potts.md

```diff
@@ -0,0 +1,104 @@
+---
+title: Potts implementation
+nav_order: 2
+parent: Implementation setup
+---
+
+# Potts implementation setup
+{: .no_toc }
+
+## Table of Contents
+{: .no_toc .text-delta }
+
+- TOC
+{:toc}
+
+The `potts` tag describes Cellular Potts Model (CPM) framework-specific setup.
+Nested tags include `potts.term` for Hamiltonian terms and `potts.parameter` for parameters.
+
+```xml
+<potts>
+    <potts.term id="[ID]" />
+    <potts.parameter id="[ID]" value="[VALUE]" scale="[SCALE]" />
+    <potts.parameter id="[ID]" value="[VALUE]" scale="[SCALE]" term="[TERM]" />
+    <potts.parameter id="[ID]" value="[VALUE]" scale="[SCALE]" term="[TERM]" target="[TARGET]" />
+    ...
+</potts>
+```
+
+## Hamiltonian terms
+
+The `potts.term` tag is used to specify the terms used in the CPM Hamiltonian.
+Valid options include:
+
+`volume`
+: Target area (2D) or volume (3D) constraint
+
+`adhesion`
+: Adhesion contact energy (2D and 3D)
+
+`substrate`
+: Differential substrate adhesion contact energy (3D only)
+
```

Comment:
Are `surface` and `persistence` also valid potts.term?

Reply:
Yes, added. I also added `height` but not `junction` (we need revisit/rename/remove that one, so leaving it out of the docs for now)